.github/workflows/ci-dgo-tests.yml

```diff
@@ -0,0 +1,59 @@
+name: ci-dgo-tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - master
+  schedule:
+    - cron: "0 4 * * *" # daily at 4:00
```

Comment:
nitpick - make this `0 0 * * *` the current config would be 4am UTC

Reply:
changing to 0 0 * * *